dali/operators/signal/decibel/to_decibels_op.h

```diff
@@ -12,17 +12,25 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef DALI_OPERATORS_SIGNAL_DECIBEL_TO_DECIBELS_H_
-#define DALI_OPERATORS_SIGNAL_DECIBEL_TO_DECIBELS_H_
+#ifndef DALI_OPERATORS_SIGNAL_DECIBEL_TO_DECIBELS_OP_H_
+#define DALI_OPERATORS_SIGNAL_DECIBEL_TO_DECIBELS_OP_H_
 
 #include <cmath>
+#include <memory>
 #include <string>
 #include <vector>
 #include "dali/core/common.h"
 #include "dali/kernels/kernel_manager.h"
 #include "dali/kernels/signal/decibel/to_decibels_args.h"
 #include "dali/pipeline/operator/common.h"
 #include "dali/pipeline/operator/operator.h"
+#include "dali/pipeline/util/operator_impl_utils.h"
+
+#define TO_DB_SUPPORTED_TYPES (float)
```

Comment:
I wonder if we need this kind of indirection for one type... And couldn't we support doubles on CPU?